ffwd-http-reporter/src/test/java/com/spotify/metrics/ffwd/FastForwardHttpReporterTest.java

```diff
@@ -45,6 +45,7 @@
     private HttpClient httpClient;
 
     private Map<String, String> commonTags;
+    private Map<String, String> commonResource;
```

Comment:
yeah i was conflicted between trying to make things match this or not: https://github.com/spotify/ffwd-http-client/pull/2/files#diff-c826143dc23720bac8fc6006a1cde221R39

Reply:
hmm yeah I hate those ones. I'd lean towards the plural so that this source file/repo is at least consistent - it has a higher cohesiveness-factor (not that that's actually a thing) than this does to a different repo.